editor/animation_editor.cpp

```diff
@@ -3700,7 +3700,7 @@ void AnimationKeyEditor::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_menu_track"), &AnimationKeyEditor::_menu_track);
 	ClassDB::bind_method(D_METHOD("_clear_selection_for_anim"), &AnimationKeyEditor::_clear_selection_for_anim);
 	ClassDB::bind_method(D_METHOD("_select_at_anim"), &AnimationKeyEditor::_select_at_anim);
-	ClassDB::bind_method(D_METHOD("_track_pos_draw"), &AnimationKeyEditor::_track_pos_draw);
+	ClassDB::bind_method(D_METHOD("_track_bar_draw"), &AnimationKeyEditor::_track_pos_draw);
```

Comment:
Would be worth changing the name of the C++ method too for clarity.

Reply:
Or maybe leave the old name, it's not a user facing method, it's just to bind a signal for the editor update. If you *really* want to change it, you should also rename `track_pos` to `track_position` and `timeline_pos` to `timeline_position` etc.